app/commands/donations/subscription/update_amount.rb

```diff
@@ -0,0 +1,7 @@
+class Donations::Subscription::UpdateAmount
+  include Mandate
+
+  initialize_with :subscription, :amount_in_cents
+
+  def call = subscription.update!(amount_in_cents:)
```

Comment:
Do an insiders check here too, in case we end up having a threshold for insiders (we will do with Pro for example)